tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -501,6 +501,17 @@ def param_value(self, verbose, name, ptype, value, textfile, binfile, command):
                     # special handle the use of Python scripts in Windows environment:
                     # 1. search for a wrapper executable (of the same name) under the same directory with stat.S_IXUSR flag
                     # 2. if no wrapper is present, prepend command with 'python' executable
+                    def split_with_single_quote_enclosed_fixup(command):
```

Comment:
Since this is a single use local function I would rather keep the code inline. Also in the current patch it seems to imply that the previously existing comment above now relates to the function but that isn't the case. Therefore please move the logic above that comment.